database/migration/aruha-522-timelines.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE zn_data.storage (
+  st_id            VARCHAR(36) NOT NULL PRIMARY KEY,
+  st_type          VARCHAR(32) NOT NULL,
+  st_configuration JSONB       NOT NULL
+);
+
+
+CREATE TABLE zn_data.timeline (
+  t_id              UUID                     NOT NULL PRIMARY KEY,
+  et_name           VARCHAR(255)             NOT NULL REFERENCES zn_data.event_type (et_name),
+  t_order           INT                      NOT NULL,
+  st_id             VARCHAR(36)              NOT NULL REFERENCES zn_data.storage (st_id),
+  t_topic           VARCHAR(255)             NOT NULL,
+  t_created_at      TIMESTAMP WITH TIME ZONE NOT NULL,
+  t_switched_at     TIMESTAMP WITH TIME ZONE DEFAULT NULL,
+  t_cleanup_at      TIMESTAMP WITH TIME ZONE DEFAULT NULL,
+  t_latest_position JSONB                    DEFAULT NULL,
+  UNIQUE (et_name, t_order)
+);
+
+CREATE INDEX ON zn_data.timeline(et_name);
```

Comment:
Well, there are several things why I do not create indexes now. 
1. They are mostly not used, and sequential scan performs fast enough to deal with it. 
2. We will use timelines almost always from cache. That means that the only operation that probably should be optimized - getting all timelines for event type. 
3. I can not guarantee on this stage what kinds of queries should be optimized, therefore I made only index, that is used now (but again, for 1000 records we can go without indexes at all)